cmd/aries-js-worker/vue-aries-framework-go/src/components/HelloFrameworkGo.vue

```diff
@@ -0,0 +1,58 @@
+<!--
```

Comment:
Please remove logo.png

Reply:
Removed. However, the app should have one or two icons at some point (even if it's just a sample app) - can't imagine it being "pretty" without any.